Directory.Packages.props

```diff
@@ -50,7 +51,8 @@
     <PackageVersion Include="Remotion.TypePipe.Documentation" Version="$(TypePipeVersion)" />
     <PackageVersion Include="Selenium.Support" Version="4.35.0" />
     <PackageVersion Include="Selenium.WebDriver" Version="4.35.0" />
-    <PackageVersion Include="SkiaSharp.NativeAssets.Linux.NoDependencies" Version="3.119.0" />
+    <!-- NOTE: This package version must match the SkiaSharp package for Linux support to work. -->
+    <PackageVersion Include="SkiaSharp.NativeAssets.Linux.NoDependencies" Version="3.116.0" />
```

Comment:
Sanity check: do we actually need to specify this package explicitly or can we also remove this line?